README.md

```diff
@@ -112,6 +112,22 @@ FROM myexport
 WHERE JSON_EXTRACT_PATH_TEXT(CustomVars, 'acct_adminDisabled_bool') = 'false';
 ```
 
+## Dockerizing Hauser
+To add a specific `hauser` binary version to a Docker container (assuming that the base image is linux),
+simply add the following to the Dockerfile:
+
+```Dockerfile
+RUN curl -L >hauser.tar.gz https://github.com/fullstorydev/hauser/releases/download/v${HAUSER_VERSION}/hauser_${HAUSER_VERSION}_linux_x86_64.tar.gz \
+  && tar -xzvf hauser.tar.gz -C /usr/bin \
+  && rm hauser.tar.gz
+```
+Where `${HAUSER_VERSION}` is the desired release version.
+(This value can also be provided at build time with the [docker ARG command](https://docs.docker.com/engine/reference/builder/#arg).)
+
+For an example of a dockerized hauser solution that dynamically creates the build config files from a template,
+see [this recipe](./recipes/multi-hauser/README.md).
+
+
```

Comment:
@claygoddard @van-fs Just added this per the comment about closing Schuylr's PR. Does this seem like a reasonable "solution", or do you think we need to have more guidance for a single hauser deployment?

Reply:
I find the above helpful.  I word-smithed it below to be a bit more prescriptive for comparison.  I feel like if we needed to be any more specific, we'd just defer to the multi-instance Hauser container.  The only difference between single and multi is the number of lines in the CSV.  (Obviously, there are technical implications, but from a narrative, it seems more understandable to have a single point of view versus two).  Long way of saying, short and sweet.

## Dockerizing Hauser
To include Hauser in a Docker container, add the following to your Dockerfile. Note that the example below assumes the base image is linux.

```Dockerfile
RUN curl -L >hauser.tar.gz https://github.com/fullstorydev/hauser/releases/download/v${HAUSER_VERSION}/hauser_${HAUSER_VERSION}_linux_x86_64.tar.gz \
  && tar -xzvf hauser.tar.gz -C /usr/bin \
  && rm hauser.tar.gz
```

The `${HAUSER_VERSION}` can be provided at build time with the [docker ARG command](https://docs.docker.com/engine/reference/builder/#arg).  You can find the latest version of Hauser on the [releases](https://github.com/fullstorydev/hauser/releases/latest) page.  For a more complete example of dockerizing Hauser, see [this recipe](./recipes/multi-hauser/README.md).

Reply:
Ok, I updated it to use your word-smithing!